mathics/core/symbols.py

```diff
@@ -395,25 +412,8 @@ def __new__(cls, name: str, sympy_dummy=None, value=None):
             # value attribute.
             self.sympy_dummy = sympy_dummy
 
-            # This is something that still I do not undestand:
```

Comment:
When you don't understand, open a discussion or QA for this, if I created this. Thanks

Reply:
If I would have do that, then I would be even more lost about what are these attributes about.